tests/functional/fake-amo-server.js

```diff
@@ -89,7 +90,7 @@ http
       process.exit(1);
     }
   })
-  .listen(8989, '127.0.0.1', () => {
+  .listen(8989, os.platform() === 'win32' ? '::1' : '127.0.0.1', () => {
```

Comment:
could we exclusively use ipv6 here?

Reply:
that does unfortunately depend from how the system decide to resolve localhost, on the other ci workers it seems to be resolved into the ipv4 address, and only the win32 one to be resolving it to the ipv6 one.

Reply:
can we pass `'localhost'` then? I am wondering if we could simplify here.

Reply:
ah, yeah, good point, that's definitely worth a shot :ok_hand:

Reply:
Great, using 'localhost" on all platform worked as we expected.